CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+### How to make a clean Pull Request:
+
+1. Make sure your pull request has a correct title. For example in PlatformUI we use 'EZP-XXXXX: Title of the pull request'
+2. In the description of the pull request:
+⋅⋅* First add the JIRA link of your story.
+⋅⋅* Then add a short description about what's in the pull request. If it's a bugfix explain why and how it fixes the issue.
+⋅⋅* Sometimes a screenshot, or a screencast is a good way to show your work.
+⋅⋅* Don't forget to mention if your code is tested (manual and unit tests, coverage...).
+3. Make a last check of the code you've just pushed, for example you can check again:
```

Comment:
You added `again` but didn't explain it before.

So the first part of this doc should explain to create a branch containing the issue number.
Say that you should run unit test locally as explained in the README.md